extensions/amp-ima-video/0.1/test/test-amp-ima-video-element.js

```diff
@@ -0,0 +1,58 @@
+/**
```

Comment:
The existing file `test-amp-ima-video.js` is a misnomer since it does not actually test the `<amp-ima-video>` element, only what it loads inside the frame. (#34230) A new `test-amp-ima-video-element.js` test is added to test the component. It currently only tests that the children element are serialized as specified previously.